timescaledb/overview/release-notes/index.md

```diff
@@ -1,43 +1,67 @@
-# TimescaleDB Release Notes & Plans
+# TimescaleDB Release Notes & Future Plans
 
-Interested in what's coming down the pipeline? Read about our
-[current plans](#current-plans). Interested in learning more about
-what's already available? Jump down below to review our
-[release notes](#release-notes).
+Interested in what's coming down the pipeline? Review our
+Future Plans section. Interested in learning more about
+what's already available? Jump down below to see what's
+been released.
 
-Want to learn more? We welcome you to visit our
-[Github repo][github] or join our [Slack community][slack].
+## Future Plans
 
-## Current Plans [](current-plans)
+TimescaleDB is an open-source project with a vibrant community. 
+We are currently focusing on making our priorities known by that community; 
+we welcome you to visit our Github repo or join our [Slack community](https://slack.timescale.com).
+
+### What to expect from our next release
+
+For our next release, we plan to add:
+
+- Compression policies on distributed hypertables
+- Partially mutable compressed chunks to support INSERTs into a compressed hypertable.
+- Various bug fixes.
+
+The current GA (Generally Available) version is 2.2.1.
 
-One of the most significant changes in TimescaleDB 2.0 is support for
-horizontally scale-out TimescaleDB running across many nodes.
 You can read more about our architecture and design for distributed hypertables
-[here][multinode-intro], and follow [these instructions][multinode-setup] to
-setup your own multi-node TimescaleDB cluster.
+[here][distributed-hypertables].
 
-### Latest release
+If you have questions about distributed hypertables, join our #multinode channel on
+[community slack](https://slack.timescale.com/) for installation details and
+follow these [setup instructions][distributed-hypertables-setup].
 
-Currently, the latest Generally Available (GA) release is TimescaleDB 2.1.
+In addition to multi-node, we've also reassessed how some core
+functionality works, and, as a result, made APIs simpler and more consistent,
+while also empowering users with more control and flexibility to customize
+behaviors to suit your needs.  Some of these API updates are **breaking changes**.
 
-What's new in TimescaleDB 2.1:
+### What's new in TimescaleDB 2.2.1:
+
+Skip Scan optimization on single node and multinode. This feature offers significant 
+improvements in performance of `SELECT` queries with `DISTINCT ON`.
+
+This release also adds support for creating a distributed 
+restore point. This allows performing a consistent restore of a 
+multi-node cluster from a backup.
```

Comment:
Nit: In general, I think plural form flows better. Feel free to ignore if you want.
```suggestion
This release also adds support for creating distributed 
restore points. This allows performing consistent restores of 
multi-node clusters from a backup.
```